packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -592,6 +608,15 @@ export interface ProjectProps extends CommonProjectProps {
    * @see https://docs.aws.amazon.com/codebuild/latest/userguide/sample-multi-in-out.html
    */
   readonly secondaryArtifacts?: IArtifacts[];
+
+  /**
+   * Support the batch build type.
+   *
+   * Set to `true` if you want to be able to run this project as a batch build.
+   *
+   * @default false
+   */
+  readonly supportBatchBuildType?: boolean;
```

Comment:
> I left the supportBatchBuildType option in for now because I think it's something you also might want to enable for triggering a build manually? Otherwise you'd need to construct the project and then immediately call enableBatchBuilds().

I don't like this property, because it means there will be 2 ways to do the same thing. I don't see a problem with calling `enableBatchBuilds()` on a Project instance.

I would get rid of this.

Reply:
Fair enough. Will remove that option.